submissions/aymkin/OOP-frogger/app.js

```diff
@@ -0,0 +1,121 @@
+let scoreCount = 0;
+
+const START = {
+	x: 202,
+	y: 405,
+};
+
+const FIELD = {
+	column: 101,
+	row: 83,
+	leftBorder: 101,
+	rightBorder: 303,
+	topBorder: 0,
+	bottomBorder: 322,
+};
+
+const SPEED = (min = 75, max = 200) =>
+	Math.floor(Math.random() * (max - min + 1)) + min;
+
```

Comment:
`SPEED()` - is a method for Enemy class. So, we can define them in class. Also, methods must be named with a verb, like so `getEnemySpeed()`.

Reply:
done those two conditions